bin/mainflux-docker.sh

```diff
@@ -81,13 +81,13 @@ _start() {
   printf "\nWaiting for Cassandra to start. This takes time, please be patient...\n"
 
   # Wait until Cassandra is ready to accept cqlsh commands
-  # or timeout after 15 sec
+  # or timeout after 100 sec
   c_on=0
   retries=0
-  while [[ $retries -lt 15 ]]
+  while [[ $retries -lt 100 ]]
   do
     # Check if C* port 9042 is open
-    if $(docker exec -it mainflux-cassandra cqlsh -e "exit" > /dev/null 2>&1)
+    if $(docker exec -i mainflux-cassandra cqlsh -e "exit" > /dev/null 2>&1)
```

Comment:
How does not having TTY affects Linux execution? If there is no difference, then this change is OK with me. Although I am not sure that we assure Windows support by design - Mainflux is developed and deployed on Unix machines, there is no guarantee that it will work on Windows.